pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -13,15 +23,19 @@ int socket__http_filter(struct __sk_buff* skb) {
         return 0;
     }
 
+    // don't bother to inspect packet contents when there is no chance we're dealing with plain HTTP
+    if (!(skb_info.tup.metadata&CONN_TYPE_TCP) || skb_info.tup.sport == HTTPS_PORT || skb_info.tup.dport == HTTPS_PORT) {
```

Comment:
We definitely don't need to worry about this just yet, but it may be worth noting that if we eventually want to support HTTP/3 then we won't be able to filter out UDP messages here